### PR TITLE
UX-606/updates to cluster details page

### PR DESCRIPTION
**Pull request by jruddell**
- header card
- tabs at top
- back link inline
- fixed polling always showing the spinner

image

______

- node list 
- table design

image

______

- info card styles

image

### CODE REVIEW

File: src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
 import React from 'react'
 import { Dialog, DialogTitle, DialogContent } from '@material-ui/core'
 import { makeStyles } from '@material-ui/styles'
 import CloseIcon from '@material-ui/icons/Close'
 import clsx from 'clsx'
 import ExternalLink from 'core/components/ExternalLink'
 import ClusterStatusSpan from 'k8s/components/infrastructure/clusters/ClusterStatus'
-import createListTableComponent from 'core/helpers/createListTableComponent'
-import { noop, pathStrOr } from 'utils/fp'
+// import createListTableComponent from 'core/helpers/createListTableComponent'
```
Comment (kevinylee1991) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js:9]: comment still needed?

File: src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js [hunk 2]
```diff
@@ -121,19 +154,19 @@ const statusMap = new Map([
...
   ['ok', 'Completed'],
   ['loading', 'In Progress'],
-  ['retrying', 'Failed... Retrying'],
+  ['retrying', 'Retrying'],
   ['failed', 'Failed'],
   ['disconnected', 'Unknown'],
 ])
 
-const renderStatus = (_, { status, logs }) => <RenderNodeStatus status={status} logs={logs} />
+// const renderStatus = (_, { status, logs }) => <RenderNodeStatus status={status} logs={logs} />
```
Comment (kevinylee1991) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js:162]: comment still needed?

File: src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js [hunk 3]
```diff
@@ -156,30 +189,44 @@ export const NodeTaskStatus = ({ status, iconStatus = false, children }) => {
...
+    <div className={classes.taskRow}>
+      <RenderNodeStatus status={props.status} logs={props.logs} />
+      <Text variant="body2" component="span" className="task-value">
+        {props.task}
+      </Text>
+    </div>
+  )
+}
+
+// const columns = [
```
Comment (kevinylee1991) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/TaskStatusDialog.js:217]: comment still needed?

**Review by kevinylee1991 — APPROVED** [review]
Changes seem fine to me. 1 thing I noticed is that the Unknown icon looks off centered

**Comment by jruddell** [comment]
> Changes seem fine to me. 1 thing I noticed is that the Unknown icon looks off centered

Yea I believe I addressed that but didn't update the screenshot. let me verify before merging.

**Pull request merged**